src/components/MoreInfo/MoreInfo.component.js

```diff
@@ -6,56 +6,57 @@ import {
     Description,
     Features,
     ImgContaiter,
-    InfoImg,
     OtherImgs,
-    SmallImg,
-    SmallImgCont
+    MainImage,
+    SmallImages,
 } from "./MoreInfo.style";
 import {BigTitle, MiddleTitle} from "../../common/Titles/Titles";
+import MoreInfoImages from "./MoreInfoImages/MoreInfoImages.component";
 
 
-const MoreInfo = ({closeMoreInfo, product}) => {
+const MoreInfo = (props) => {
+    const {closeMoreInfo, product} = props
+
+    const {author,
+        producer, price,
+        features, title,
+        conditions, rating,
+        gunres, description} = product
+
```

Comment:
or one row
or every var name in new row
Because nobody won't work here near feature, but will read it.

Reply:
fixed. made every var in a new line.